frontend/src/services/ApiServiceFactory.ts

```diff
@@ -24,7 +23,6 @@ type AppAccessControlApiType = {
 
 type AdminManagementApiType = {
     applicationAdminApi: FAMApplicationAdminApi;
-    applicationsApi: FAMApplicationsApi;
```

Comment:
removed it, as we no longer use it anymore

Reply:
Thanks Catherine!
Would you mind also remove the comment on line 11, but maybe we can still keep the "alias".

Reply:
sure, I'll do that